pyonfx/utils.py

```diff
@@ -43,18 +47,20 @@ def all_non_empty(lines_chars_syls_or_words):
         return out
 
     @staticmethod
-    def clean_tags(text):
+    def clean_tags(text: str):
         # TODO: Cleans up ASS subtitle lines of badly-formed override. Returns a cleaned up text.
         pass
 
     @staticmethod
-    def accelerate(pct, accelerator):
+    def accelerate(pct: float, accelerator: float):
```

Comment:
```suggestion
    def accelerate(pct: float, accelerator: float) -> float:
```

Reply:
Adding return type here might not be necessary, as its return type is inferred from its `return` statement:

![image](https://user-images.githubusercontent.com/7723097/108404476-7e7df200-7252-11eb-85f7-e4856eb0c6e9.png)